README.md

```diff
@@ -31,6 +32,7 @@ Fedora versions currently supported are:
 * Fedora 27
 * Fedora 28
 * Fedora 29
+* Fedora 30
```

Comment:
Sorry, I am a nitpicker. I guess, Only Fedora 30 is supported, right?

Reply:
I'd rather remove outdated version in a different PR to keep this one clean.

golang 1.13 would add F31 to the list - and another PR to drop outdated versions (if these are no longer supported) - does that sound good?